src/ldp/representation/RepresentationMetadata.ts

```diff
@@ -159,13 +160,18 @@ export class RepresentationMetadata {
    * @param subject - Subject of quad to add.
    * @param predicate - Predicate of quad to add.
    * @param object - Object of quad to add.
+   * @param graph - Optional graph of quad to add.
    */
   public addQuad(
     subject: NamedNode | BlankNode | string,
     predicate: NamedNode | string,
     object: NamedNode | BlankNode | Literal | string,
+    graph?: MetadataGraph,
   ): this {
-    this.store.addQuad(toSubjectTerm(subject), toCachedNamedNode(predicate), toObjectTerm(object, true));
+    this.store.addQuad(toSubjectTerm(subject),
+      toCachedNamedNode(predicate),
+      toObjectTerm(object, true),
+      graph ? toSubjectTerm(graph) : undefined);
```

Comment:
(minor) `toSubjectTerm` might be misnomer if it's `DefaultGraph`.

Reply:
`toNamedTerm`? Didn't want to make a similar function for graphs since it's a bunch of overloads for a function that does pretty much the same.

Reply:
`toTerm`? Either is fine with me.